src/VisualStudio/Core/Def/Implementation/Utilities/VisualStudioNavigateToLinkService.cs

```diff
@@ -0,0 +1,39 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System;
+using System.Composition;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.CodeAnalysis.Editor;
+using Microsoft.CodeAnalysis.Host.Mef;
+using Roslyn.Utilities;
+
+namespace Microsoft.VisualStudio.LanguageServices.Implementation.Utilities
+{
+    [ExportWorkspaceService(typeof(INavigateToLinkService), layer: ServiceLayer.Host)]
+    [Shared]
+    internal sealed class VisualStudioNavigateToLinkService : INavigateToLinkService
+    {
+        [ImportingConstructor]
+        [Obsolete(MefConstruction.ImportingConstructorMessage, error: true)]
+        public VisualStudioNavigateToLinkService()
+        {
+        }
+
+        public Task<bool> TryNavigateToLinkAsync(Uri uri, CancellationToken cancellationToken)
+        {
+            if (!uri.IsAbsoluteUri)
+            {
+                return SpecializedTasks.False;
```

Comment:
is this just a temp limitation?  Or is the caller expected to have absolutized things first.  If so, should we doc that that's an expectation?

Reply:
➡️ Will file a follow-up issue.

Reply:
➡️ Filed follow-up issue #37501